stack/base-with-services/Dockerfile

```
@@ -25,3 +25,10 @@ WORKDIR "/home/${NB_USER}"
 
 # Initialize the database
 RUN mamba run -n aiida-core-services initdb -D aiida_db -U aiida
+
+# Install Quantum ESPRESSO and its AiiDA plugin
+RUN mamba install qe
+RUN mamba install aiida-quantumespresso==4.0.1
```

I would suggest to run this in one step:
```suggestion
RUN mamba install \
    aiida-quantumespresso==4.0.1 \
    qe \
  && mamba clean --all -f -y && \
  fix-permissions "${CONDA_DIR}"
```
Also, would be prudent to specify the desired version of `qe`.